.github/actions/setup-nextest/setup.sh

```
@@ -0,0 +1,35 @@
+	;;
+aarch64)
+	# No prebuilt musl binary for ARM, and glibc binary doesn't work with gcompat
+	# (__res_init symbol missing). Build from source instead.
+
+	# Install build dependencies (apk add is idempotent with --no-cache)
+	apk add --no-cache gcc musl-dev
+
+	# Install Rust if not present (minimal profile for faster install)
+	if ! command -v cargo >/dev/null 2>&1; then
+		curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile minimal
+		. "$HOME/.cargo/env"
```

Shellcheck (0.11.0)

[info] Not following: ./.cargo/env was not specified as input (see shellcheck -x).

(SC1091)